lib/ex_docker_build.ex

```diff
@@ -201,6 +201,39 @@ defmodule ExDockerBuild do
     end
   end
 
+  @spec inspect_volume(String.t()) :: :ok | {:error, any()}
+  def inspect_volume(volume_name) do
+    case DockerRemoteAPI.inspect_volume(volume_name) do
```

Comment:
you are inspecting a volume but only returning `:ok`, what happened with the body? you should be returning it, also where are the tests?